Makefile.am

```diff
@@ -152,6 +152,20 @@ libmetal_a_SOURCES = \
 	src/vector.S \
 	src/watchdog.c
 
+########################################################
+# libpico_metal
+########################################################
+
+lib_LIBRARIES += libriscv__pico__metal.a
```

Comment:
Do we want to wrap this library in an option WITH_BUILTIN_LIBPICO_GLOSS or something similar, like we do for libgloss?

Also, is it worth matching the naming convention below more with a name like `libmetal-pico-gloss.a`?

Reply:
In your Freedom-E-SDK pull request you name it `libmetal-pico.a`, so we can match that name here instead of moving the file in that Makefile.

Reply:
done